### DISCUSSION TITLE
Global Plugins

### DISCUSSION THREAD

**Discussion by Sharaf5** [godotengine/godot-proposals#7291] [Engine Core]
I see godot is managing plugins per project
However this seems to be not ideal as godot compiles result to other formats

**What I suggest** is similar to GDScript inhiritance but for plugins
a global path to load plugins from, unless the same is in local project (either modified or not) 
- Godot should use local scripts and local plugins as normal and should have access to plugin global path
- user **must** enable the global plugin in project to load it
- any plugin found in both local and global plugins should be scaned to see resource matches
- Godot deside form where to load plugin resources like script class inhiritance but only load one resource of two matches
- modifing global plugin resources affects all projects loading it but local only override global specific resource
- any global plugin could be entirely made local by clonnig / copping full dir to local and deactivating it from global plugins

Why I'm in disscussion not in issues
I'm a new commer to the world of game dev, and this may be a feature already exists but I don't know about

**Comment by AThousandShips** [comment]
* Duplicate of: #831

Please add your support and thoughts to that instead
